tensorflow/python/ops/logging_ops.py

```diff
@@ -176,9 +171,8 @@ def print_v2(*inputs, **kwargs):
 
     Usage in a defun:
     ```python
-    tf.enable_eager_execution()
-
-    @tf.contrib.eager.defun
+    from tensorflow.python.eager import function
```

Comment:
tf.function exists so no need to import from contrib which doesn't exist in tf2

Reply:
`from tensorflow.python.eager import function` is for decorator function.defun.
refer to [`control flow unit test`](https://github.com/tensorflow/tensorflow/blob/1d54cbf4a2252215c5d2ce9accb5e498a7c2a704/tensorflow/python/kernel_tests/control_flow_util_v2_test.py)

Reply:
We need to use the public API in our documentation.

Reply:
```python
from tensorflow.python.framework import function
@function.Defun()
def f():
  tensor = tf.range(10)
  tf.print(tensor, output_stream=sys.stderr)
  return tensor

range_tensor = f()
```
raise InvalidArgumentError as is expected

Reply:
Yes but don't use the deprecated function.Defun in documentation